machineconfiguration/v1/helpers.go

```diff
@@ -0,0 +1,198 @@
+package v1
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewMachineConfigPoolCondition creates a new MachineConfigPool condition.
+func NewMachineConfigPoolCondition(condType MachineConfigPoolConditionType, status corev1.ConditionStatus, reason, message string) *MachineConfigPoolCondition {
```

Comment:
hm.. the helper functions usually end up in library-go instead of the API..

Reply:
looks like they're not used anywhere else and they could stay in the MCO repo for now.